source/normalize-arguments.ts

```diff
@@ -436,7 +436,11 @@ export const normalizeRequestArguments = async (options: NormalizedOptions): Pro
 	// `process.version.electron` is used only once, right here.
 	if (options.useElectronNet && (process.versions as any).electron) {
 		const electron = dynamicRequire(module, 'electron') as any; // Trick webpack
-		options.request = electron.net.request ?? electron.remote.net.request;
+		options.request = deprecate(
+			electron.net.request ?? electron.remote.net.request,
+			'Electron support has been deprecated and will be removed in Got 11',
```

Comment:
Can you add a link to https://github.com/sindresorhus/got/issues/899? So people can easily know why we are doing this.